cmd/registry/push/push_test.go

```diff
@@ -99,7 +99,7 @@ Flags:
       --platform stringArray       os and architecture of the artifact in OS/ARCH format (only for plugins artifacts)
   -r, --requires stringArray       set an artifact requirement (can be specified multiple times). Example: "--requires plugin_api_version:1.2.3"
   -t, --tag stringArray            additional artifact tag. Can be repeated multiple times
-      --type ArtifactType          type of artifact to be pushed. Allowed values: "rulesfile", "plugin"
+      --type ArtifactType          type of artifact to be pushed. Allowed values: "rulesfile", "plugin", "asset"
```

Comment:
It would be nice to implement the flag using the enum here: https://github.com/falcosecurity/falcoctl/blob/main/pkg/options/enum.go

Reply:
oci.ArtifactType already implements the cobra `Value` interface. I prefer this way instead of a custom one, if I understood correctly

Reply:
The `enum` type would be more suitable for such a use case. Up to you if you want to implement it or not.

Reply:
I will introduce this in a second PR to speed up this one entering the codebase